docs/releases/4.2.8.txt

```diff
@@ -11,3 +11,5 @@ Bugfixes
 
 * Fixed a regression in Django 4.2 that caused :option:`makemigrations --check`
   to stop displaying pending migrations (:ticket:`34457`).
+* Fixed silent ``MultiPartParser`` failures for when the chunks' header fields
+  are larger than 1024 bytes.
```

Comment:
The issue has been present since the feature was introduced. Per our backporting policy this means it doesn't qualify for a backport to 5.0.x or 4.2.x anymore.

Please remove a release note.

Reply:
Will reluctantly adjust... the fact that it has been failing silently sounds like something that I'd see as valuable to fix as soon as feasible.